Minor edit to Embed best practices (#6718)
GoogleChrome/web.dev: src/site/content/en/blog/embed-best-practices/index.md

Third-party content can impact the performance of a page in many ways. It can be render-blocking, contend with other critical resources for network and bandwidth, or affect the Core Web Vitals metrics. Third-party embeds may also cause layout shifts as they load. This article discusses performance best practices that you can use when loading third-party embeds, efficient loading techniques, and the Layout Shift Terminator tool that helps reduce layout shifts for popular embeds.

{% Aside %}

> It would be best to use the techniques described in this post to load only offscreen or non-primary page content. This would ensure that all critical content gets indexed by Google.

Reviewer: ```suggestion
It would be best to use the techniques described in this post to load only offscreen or non-primary page content. This would ensure that all critical content gets indexed by search engines.
```

Author: Committed suggested change with a link to search docs.

> It would be best to use the techniques described in this post to load only offscreen or non-primary page content. This would ensure that all critical content gets indexed by [search engines](https://developers.google.com/search/docs/advanced/javascript/lazy-loading).

Reviewer 2: ```suggestion
It's best to use the techniques described in this post to load only offscreen or non-primary page content. This ensures that all the critical content gets indexed by [search engines](https://developers.google.com/search/docs/advanced/javascript/lazy-loading).
```
